WebHostLib/customserver.py

```diff
@@ -20,6 +20,8 @@
 
 from MultiServer import Context, server, auto_shutdown, ServerCommandProcessor, ClientMessageProcessor, load_server_cert
 from Utils import restricted_loads, cache_argsless
+
+from . import app
```

Comment:
```suggestion
```
This seems to be a dead import. Any import in custom server has to be double-checked, because loading the wrong stuff will make our memory use explode. Since this wasn't picked up by any other reviewer, I am now wondering if we need a checklist for custom server?